### PR TITLE
scenario 4 complete

### PR DESCRIPTION
**Pull request by sabinala**
See scenario 4 notebook.

### CODE REVIEW

File: 18-month-milestone/hackathon/epi/.ipynb_checkpoints/Scenario 4 Microbial Regnets-checkpoint.ipynb
Comment (djinnome) [inline 18-month-milestone/hackathon/epi/.ipynb_checkpoints/Scenario 4 Microbial Regnets-checkpoint.ipynb:1]: It looks like you accidentally included this file?

**Review by djinnome — APPROVED** [review]
other than the extraneous checkpoint file, looks great!

**Comment by djinnome** [comment]
oh, wait.  I think it makes sense to create a `scenario4` folder under which we include a subfolder `ciemss` where we include our results so that we don't mix it with other performers.

**Comment by sabinala** [comment]
But isn't this the ciemss forked version of the repo?

**Comment by sabinala** [comment]
Oh nevermind...

**Comment by sabinala** [comment]
I think it's ok to leave it for now. Others have the option of naming the notebook something else, and ultimately this will need to be done in Terarium.

**Pull request merged**